server/container/guice/blob-objectstorage-guice/src/main/java/org/apache/james/modules/objectstorage/ObjectStorageDependenciesModule.java

```diff
@@ -44,7 +44,7 @@
 
 public class ObjectStorageDependenciesModule extends AbstractModule {
 
-    private static final String OBJECTSTORAGE_CONFIGURATION_NAME = "objectstorage";
+    public static final String OBJECTSTORAGE_CONFIGURATION_NAME = "objectstorage";
```

Comment:
this file is related to swift/S3, you can reuse it elsewhere.
If you want the same file to be used for several things, you have to use a name declared elsewhere

Reply:
I remember, we had a discussion and came to the choice of using only one configuration file for whole things related to blobstore. That means, put all configuration on one  file.

May be the file name is not really good. But using one file is final decision. You was not there in the discussion.

Reply:
Let's do `blobstore.properties` + `objectstorage.properties`